docs/docsource/Installation/configure/adding_NFS_storage.rst

```diff
@@ -273,8 +273,8 @@ Installing the Chart with a Standalone Cloudserver Instance
 Manually Trigger Sync
 ---------------------
 
-This chart deploys a Kubernetes Job at install to immediately begin a metadata
-sync. Additionally, a Kubernetes CronJob is deployed, which periodically
+This chart deploys a Kubernetes job at install to immediately begin a metadata
```

Comment:
The "Kubernetes Job" is another example of this.

Reply:
I hate the way Google's tech writers seem to think capitalizing things ungrammatically makes them clearer.

Another Job object.